setup.py

```diff
@@ -245,8 +244,7 @@ def check_submodules():
     if not SKIP_RUNTIME:
         check_submodules()
         ext_modules.extend([
-            CMakeExtension("intel_extension_for_transformers.neural_engine_py", "intel_extension_for_transformers/llm/runtime/deprecated/"),
-            CMakeExtension("intel_extension_for_transformers.llm.runtime.graph.mpt_cpp", "intel_extension_for_transformers/llm/runtime/graph/"),
+            #CMakeExtension("intel_extension_for_transformers.neural_engine_py", "intel_extension_for_transformers/llm/runtime/deprecated/"),
```

Comment:
pls reserve neural-engine because customers need it to run some models like Bert, SD...

Reply:
cc @a32543254

Reply:
Yes, we will keep deprecated in ITREX